trieste/models/keras/architectures.py

```diff
@@ -324,6 +324,9 @@ def _gen_output_layer(self) -> tf.keras.Model:
 
     def call(self, inputs: tf.Tensor) -> tf.Tensor:
 
+        if inputs.shape.rank == 1:
+            inputs = tf.expand_dims(inputs, axis=-1)
+
```

Comment:
are we not sure we would rather catch this as an error, rather than be lenient and just expand it internally?